src/Storage/Storage.php

```diff
@@ -21,6 +21,8 @@ class Storage
 
     const DEVICE_LINODE = 'linode';
 
+    const CLOUDFARE_R2 = 'cloudflare-r2';
```

Comment:
We don't use dashes for any of the others (DO Spaces) so let's not do it here.

```suggestion
    const CLOUDFLARE_R2 = 'cloudflarer2';
```